pandas/tests/io/test_pytables.py

```diff
@@ -4999,6 +4999,62 @@ def test_read_nokey_empty(self):
             store.close()
             pytest.raises(ValueError, read_hdf, path)
 
+    # GH10143
+    def test_walk(self):
```

Comment:
put the comment inside the test function

Reply:
done